docs/community/content/contribute/release.cn.md

```diff
@@ -135,7 +135,8 @@ https://github.com/apache/incubator-shardingsphere/blob/master/RELEASE-NOTES.md
 创建`${RELEASE.VERSION}-release`分支，接下来的操作都在该分支进行。
 
 ```shell
-git clone -b ${branch} https://github.com/apache/incubator-shardingsphere.git ~/incubator-shardingsphere
+## ${name}为源码所在分支，如：master，dev-4.x
+git clone --branch ${name} https://github.com/apache/incubator-shardingsphere.git ~/incubator-shardingsphere
```

Comment:
should add --depth=1?

Reply:
`--depth=1` will lost history commits, it's not convenient for release-manager to check the download branch is right or not